api/global.go

```diff
@@ -20,36 +21,56 @@ var ApmHubPath string
 var Kubernetes kubernetes.Interface
 var Namespace string
 
+// type alias because the name "Context" collides with gocontext
+// and embedding both wouldn't have been possible.
+type EchoContext = echo.Context
+
 type Context struct {
+	EchoContext
 	gocontext.Context
-	DB         *gorm.DB
+	db         *gorm.DB
```

Comment:
made it private and added `DB()` method instead which will return the db with context wrapped.